scripts/automator/build/clang-defaults

```diff
@@ -5,7 +5,7 @@ cxx="clang++${postfix}"
 # Flag additions
 TYPES+=(debug warnmore profile)
 cflags_release=("${cflags[@]}" -Os)
-cflags_debug=("${cflags[@]}" -g -Og -fno-omit-frame-pointer)
+cflags_debug=("${cflags[@]}" -g -fno-omit-frame-pointer)
```

Comment:
Even though GCC and Clang define `Og` as a debug-friendly level of optimization, I still found that many stack layers were being optimized away.  So this fixes that and improves debug-ability.